sormas-ui/src/main/java/de/symeda/sormas/ui/samples/AbstractSampleForm.java

```diff
@@ -151,7 +152,8 @@ protected void addCommonFields() {
 		addField(SampleDto.SHIPPED, CheckBox.class);
 		addField(SampleDto.RECEIVED, CheckBox.class);
 
-		addField(SampleDto.PATHOGEN_TEST_RESULT, ComboBox.class);
+		ComboBox testResultField = addField(SampleDto.PATHOGEN_TEST_RESULT, ComboBox.class);
+		testResultField.removeItem(PathogenTestResultType.NOT_DONE);
```

Comment:
Why did you remove `PathogenTestResultType.NOT_DONE` here?

Reply:
Because it's removed in every occasion PathogetTestType in a ComboBox